### PR TITLE
ImageClassificationEfficientNet

### PR DESCRIPTION
**Pull request by Shikiz**

### CODE REVIEW

**Comment by Thilakraj1998** [comment]
the accuracy is quite low for a transfer learning model
try increase number of epochs and  change optimization functions .
 you out layer has wrong number of neurons.
you have multiclass classification problem with 3 class but the output has 2 neurons specified to it.


Reason:
for binary classification usually we utilize sigmoid has activation funciton in output layer. while for multiclass it is mostly softmax which return probability of each class available thats why it is required to specify output neurons to be equal to number of classes/target label available.

In your case it will be model.add(layers.Dense(3, activation=output_activation))

**Comment by Thilakraj1998** [comment]
Resulting confusion matrix is still underwhelming
if you observe the all the test data is consider to be cat which state model is not well trained.
Before creating confusion matrix do check considering check output of y_pred to understand it.

Because result of classification report and Confusion matrix doesnt match anyways.

Also add gpu tag similar to CNN template in meta description
